try:
    4/2
except:
    print('오류가 발생합니다.')
else:
    print('오류가 없습니다.')
    
# 오류가 없을 때 else문이 실행됩니다.